update WalletGeneration to 2.0 (#1129)
zkSNACKs/WasabiDoc: docs/using-wasabi/WalletGeneration.md

## Generating the wallet step-by-step

1. Launch Wasabi Wallet.
The very first time you run the software the `Add Wallet` dialog will be open automatically, but you can also access it by clicking on `Add Wallet` in the NavBar or Searchbar.

2. Click `Create a new wallet` to generate a new wallet.

3. Name the new wallet precisely to ensure a proper differentiation from wallets created in the future.
This label is not shared with anyone, it is only stored locally on your computer.
Click `Continue`.

> 4. Write a long and random password and **[back it up](/using-wasabi/BackupBestPractices.md)**.

Reviewer: ```suggestion
4. Write a long and random password (passphrase) and **[back it up](/using-wasabi/BackupBestPractices.md)**.
```

	So, triple-check that you have a proper backup!
	:::

5. Confirm the password and click the `Continue` button.

> 6. Write down the 12 recovery words (mnemonic seed phrase).

Reviewer: ```suggestion
6. Write down the 12 recovery words (mnemonic seed phrase) in the correct order and store them in a safe place.
```

> You can use this seed phrase **together with** your password to recover your wallet on a different computer using Wasabi or another [BIP 39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) compliant wallet.

Reviewer: ```suggestion
You have to use this seed phrase **together with** your password to recover your wallet on a different computer using Wasabi or another [BIP 39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) compliant wallet.
```

![Wasabi Wallet recovery words](/WalletManagerRecoveryWords.png "Wasabi Wallet recovery words")

> 7. You must confirm 3 of the 12 Recovery Words, as a quick check to make sure that your Recovery Words are correct.

Reviewer: ```suggestion
7. You must confirm 3 of the 12 Recovery Words, as a quick check to make sure that your Recovery Words are backed up.
```

So, confirm the Recovery Words, and click `Continue`.

8. Select the coinjoin strategy for this wallet and click `Continue`.

9. The wallet is succesfully added!
Click `Done`.

10. The wallet is automatically selected.

> Enter the wallets' password to open the wallet.

Reviewer: IMO this step should be deleted because it has nothing to do with generating a wallet.

> 10. The wallet is automatically selected.
> Enter the wallets' password to open the wallet.

Reviewer 2: ```suggestion
```
